_pages/content-types/forms.md

```diff
@@ -0,0 +1,94 @@
+
+## Minimize emotional labor
+
+Understanding people’s possible attitudes toward the requests you’re making can help you determine where to provide contextual information and how much to provide. If you’ll be requesting Social Security numbers, financial data, or personal medical information, you may want to share why you need this information and explain the safeguards you’ll be taking to protect it. Providing this context can help users feel more at ease and create a more satisfying interaction. 
+
+## Design with device awareness in mind
+
+We strive to design content that’s equally accessible — and enjoyable — across all devices. That said, keeping your users’ device preferences in mind can help you make decisions about form language. If you know most people will be accessing your form on mobile, you may consider creating shorter field labels. If most people will be accessing your form in the browser, you can create longer, more descriptive field labels without compromising the user experience.  
+
+## Keep it concise
+
+Brevity is especially pertinent when creating forms. People are completing your form to achieve a certain end; don’t create more work for them by using unwieldy or long-winded descriptions. 
+
+Create labels that are concise and to the point — First name is much easier to comprehend, at a glance, than What moniker do you go by? And, as we noted earlier, short labels are also easier to read on mobile, which improves your form’s accessibility across devices and potentially saves you money on translations.
+
+## Align field length with response length
+
+Field length should correspond to the amount of information you’re requesting. Fields for capturing a first or a last name or a city of residence can be short; fields meant to capture longer responses — a complaint or request, for instance — should necessarily be longer so that people can see what they’re typing before submitting it.
+
+## Don’t use placeholder text as labels
+
+Labels should appear above fields, not as placeholder text inside those fields. Not only can using placeholder text as labels create confusion around which information belongs where (it’s easy to forget what a field is for if it’s not labeled), but it can also make form completion more difficult for people with disabilities. 
+
+## Don’t make optional information required
+
+If a piece of information is optional, label it as such. Requiring only the information that’s truly required will save your users time — something all of us can appreciate. It’s also a good practice to label optional fields instead of labeling required ones; seeing the optional labels, users will infer that the unlabeled fields are for required information. 
```

Comment:
> It’s also a good practice to label optional fields instead of labeling required ones; seeing the optional labels, users will infer that the unlabeled fields are for required information. 

The general best practice I've followed, and which I think the Web Design Standards gently nudges authors towards, is slightly different: to label *only* optional or *only* required fields depending on which is the least verbose.

If there are 4 required fields and only 1 optional field, as in the [USWDS address form example](https://standards.usa.gov/components/form-templates/#name-form), then labeling the optional one makes sense. But if there are 3 optional fields and only 2 required fields, the form may be less overwhelming if only the required ones are labeled as such, as in the [USWDS name form example](https://standards.usa.gov/components/form-templates/#name-form).